### ISSUE TITLE
NVDA does not announce jump in header

### ISSUE THREAD

**Issue by peterpacult** [nvaccess/nvda#12175]
### Steps to reproduce:
1. Open LO-Writer
2. Generate any Header
3. With "Ctrl+Picture up" you jump to the header and, if necessary, back to the text. The function is flawless; but unfortunately the screen reader does not tell me acoustically when I am in the input field of the header.

### Actual behavior:
NVDA doesn't announce the position in the "Edit Header"-Field

### Expected behavior:
NVDA should announce the position in the "Edit-Header"-Field

#### NVDA version:
2020.4
#### Windows version:
Windows
Edition	Windows 		10 Home
Version				1909
Installed on			17.02.2020
Operating system build		18363.1016

#### Other information about your system:
-
### Other questions
-
#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
-

**Comment by peterpacult** [comment]
The problem has also been reported to LibreOffice:
https://bugs.documentfoundation.org/show_bug.cgi?id=141043

**Comment by Adriani90** [comment]
cc: @michaelweghorn is there any progress on LO side with regards to the announcements when moving to header, main text or footer? Can this be implemented via iAccessible? Or should NVDA do this in its app module?

**Comment by michaelweghorn** [comment]
> cc: @michaelweghorn is there any progress on LO side with regards to the announcements when moving to header, main text or footer? Can this be implemented via iAccessible? Or should NVDA do this in its app module?

@Adriani90 In my tests with current LO and NVDA dev versions, NVDA doesn't announce the newly focused paragraph in main document or header at all when switching between header and main document.
However, at first glance, it looks to me like LO should already be reporting everything that NVDA should need, so maybe something is missing on NVDA side?

For example, when switching from the main document, NVDA apparently does get notified about the new focus, and the parent is correctly reported with a header role, as can be seen in NVDA's Python console:

```
>>> focus.role
<Role.PARAGRAPH: 47>
>>> focus.parent.role
<Role.HEADER: 70>
```

I'd assume that NVDA should be receiving a focus event and could then e.g. check whether the parent is a header or footer if it wants to announce that case in a special way.
